alembic.ini

```diff
@@ -0,0 +1,86 @@
+# A generic, single database configuration.
```

Comment:
Please move all the alembic directory structure into `app/database/alembic`

Reply:
If possible, move the configurations to `config.py.example`

Reply:
Can you please alborate why do that?
this file is runing when you call the alembic into action. this is the standart config that is well documented and it is clear to maintain it. if you whant to override things you can alwayys do it in alembic/env.py file.
but as I see it i wouldn't mix that config with the rest of the custom code configuration.
Ill be glad to hear your opinion about that.

Reply:
I'd like to avoid making the user edit multiple configuration files, but now I see that the user shouldn't really touch anything here, so I guess it's cool